contracts/Whitelist.sol

```diff
@@ -58,6 +72,17 @@ contract Whitelist is Ownable {
         emit ProductWhitelisted(productHash, _underlying, _strike, _collateral);
     }
 
+    /**
+     * @notice whitelist a collateral address, can only be called by owner
+     * @dev function can only be called by owner
+     * @param _collateral collateral asset address
+     */
+    function whitelistCollateral(address _collateral) external onlyOwner {
+        _setWhitelistedCollateral(_collateral);
```

Comment:
are we using ETH as 0x000? or are we using WETH?

Reply:
Also shouldn't be able to whitelist a collateral that is already whitelisted

Reply:
@aparnakr the checks for an already whitelisted collateral is in `_setWhitelistedCollateral()`.
For address(0) I think we can keep it this way for now and review this part when we decide.